pkg/executor/benchmark_test.go

```diff
@@ -1838,3 +1839,26 @@ func BenchmarkAggPartialResultMapperMemoryUsage(b *testing.B) {
 func BenchmarkPipelinedRowNumberWindowFunctionExecution(b *testing.B) {
 	b.ReportAllocs()
 }
+
+func BenchmarkCompleteInsertErr(b *testing.B) {
```

Comment:
How about adding this to https://github.com/pingcap/tidb/blob/91fb7d3d6b4d97bb8640b9b575054e58ef76823d/pkg/executor/union_scan_test.go#L399

So you can review the performance changes history like [this](http://107.173.155.134:18081/)

Reply:
Done, PTAL @tiancaiamao , thx!